### ISSUE TITLE
add aria fields to all link and button inputs

### ISSUE THREAD

**Issue by svsven** [equinor/energyvision#540]
Link and button fields in Sanity should have the option to add a `aria-label` so that editors can set a different, more descriptive label for screen reader users

Example:
An editor adds a teaser component and uses "Read more" as label for the link/button. This is not an accessible label as it is not at all descriptive about where the link goes. Solution: either add a more descriptive label for all users, or set a specific aria-label for screen readers.

List of places to add

- [x] Call to action | Text block
- [x] Link/action | Teaser
- [x] Link | Promo tiles
